src/main/java/com/vinylteam/vinyl/service/impl/DefaultUserService.java

```diff
@@ -103,7 +128,13 @@ public boolean update(String oldEmail, String newEmail, String newPassword, Stri
                     optionalUser = optionalUserFromDataBase;
                     if (!optionalUser.get().getStatus()) {
                         Optional<ConfirmationToken> confirmationToken = confirmationService.findByUserId(optionalUser.get().getId());
-                        confirmationToken.ifPresent(token -> confirmationService.sendMessageWithLinkToUserEmail(email, token.getToken().toString()));
+                        confirmationToken.ifPresent(token -> {
+                            try {
+                                confirmationService.sendMessageWithLinkToUserEmail(email, token.getToken().toString());
+                            } catch (Exception e) {
+                                e.printStackTrace();
+                            }
```

Comment:
Exception even is not logged, I am not sure that this try catch is needed, it should be catched on controller level